src/state.py

```diff
@@ -19,21 +20,23 @@
 WAZUH_API_CREDENTIALS = "wazuh-api-credentials"
 # Bandit mistakenly thinks this is a password
 WAZUH_CLUSTER_KEY_SECRET_LABEL = "wazuh-cluster-key"  # nosec
-WAZUH_USERS = {
-    "wazuh": {
-        "default_password": "wazuh",
-        "default": True,
-    },
-    "wazuh-wui": {
-        "default_password": "wazuh-wui",
-        "default": True,
-    },
-    # This user will be created by the charm
-    "prometheus": {
-        "default_password": "",
-        "default": False,
-    },
-}
+WAZUH_USERS = OrderedDict(
```

Comment:
Since python 3.7, dict entries order is guaranteed to be the insertion order. If you specifically need an OrderedDict, maybe you should add a comment on why (for future maintainers)